### DISCUSSION TITLE
Real Preferences Screen + Dark Mode Settings + Create Text File in Finder + Maintain OS 10.13.6 Support

### DISCUSSION THREAD

**Discussion by Leebgit** [brackets-cont/brackets#294] [Feature Request]
I clicked on Preferences to change Brackets settings to dark mode and was greeted with a bunch of HTML code. Please.

Brackets has a LOT of settings! The present way they are arranged wastes a huge amount of time for users. All I wanted to do was change the color of the text and change the background color for custom dark mode. Why make users read through 15 minutes of code to find the thing they want to change? Please, please, please make a real Preferences window with a real settings options and divide settings up based on category, like User Interface, HTML, JavaScript, PhP, Check for Updates, etc. 

I could not find settings to change colors in the settings. Please add these. I cannot believe I am the only one who'd like to customize the colors in an App I must stare at for hours at a time.

Also, it would be really great to create a function in Finder where you can right-click on a directory to create a TXT file using Brackets. This would make it easy to create notes in a programming directory and make it easier to keep track of programming details.

Finally, please maintain support for Mac OS X 10.13.6 High Sierra. Many old MacBook Pros are unable to upgrade and are stuck at this version. There is no reason to kill Brackets on these older machines.

Thank you. よろしくお願いします。

**Comment by ScanuNicco** [comment]
I think that adding a graphical settings menu is a good way to make Brackets more user-friendly, and is extra important in Phoenix where changing the settings seems to require opening "fs/app/brackets.json" after going to Debug > Open Virtual File System. That being said, settings need to be implemented in a way where it is easy to add a new setting programmatically, and I think there also need to be an API to allow extensions to create their own category within the settings menu (similar to how BetterDiscord does it). It would also be useful if an extension could prevent certain built-in from changing settings when necessary. See the attached screenshot showing how Firefox handles this:
![image](https://github.com/brackets-cont/brackets/assets/30136040/bda06dc1-70e0-4d6a-8e36-6348b56e411f)

That's just my opinion on how it should work though. Unfortunately, I don't have much time to contribute these days.

Also by the way if you want to change the colors there is a large selection of themes which you can install through the extension store.

**Comment by charlypa** [comment]
@Leebgit 
We've updated our editor to Phoenix Code, launched just yesterday. It now supports Windows, Mac, and Linux moving beyond the older Brackets x86 binaries.
You can download Phoenix Code at https://phcode.io/.
